third_party/docfx/templates/devsite/UniversalReference.common.js

```diff
@@ -25,6 +25,16 @@ exports.transform = function (model) {
         break;
       case 'module':
       case 'class':
+        if (model.langs && model.langs[0].toLowerCase() === "python" &&

```

Comment:
This seems to make it so `module` and `class` will no longer fall through below, even if you don't match this `if` statement.

Reply:
If the type is a `module` or a `class` but it does not fall under the if statement category, since there's no break outside the if statement it will fall through to below for case `interface`, `struct` or `delegate` would fall under.